docs/references/config/node_key.json.md

```diff
@@ -0,0 +1,106 @@
+---
+order: 1
+parent:
+  title: node_key.json
+  description: Description and usage of the node ID
+  order: 2
+---
+
+## The `node_key.json` file
+The node ID, the IP address of the host and the P2P port together identify a node: `nodeID@IP:port`.
+
+The easiest way to get the node ID is running the `cometbft show-node-id` command.
+
+The `node_key.json` file resides at `$CMTHOME/config/node_key.json`. This can be overridden at the
+[node_key_file](#node_key_file) parameter in the [`config.toml`](#the-configtoml-file) file.
+
+The file contains a private key (in [priv_key.value](#priv_keyvalue)) to an asymmetric algorithm
+(in [priv_key.type](#priv_keytype)).
+
+The node ID is calculated by hashing the public key with the SHA256 algorithm and taking the first 20 bytes of the
+result.
+
+### priv_key.type
+The type of the key defined under [`priv_key.value`](#priv_keyvalue).
+
+Default example in context:
+```json
+{
+  "priv_key": {
+    "type": "tendermint/PrivKeyEd25519",
+    "value": "jxG2ywUkVPiF4XDW1Dwa5ZfcrC0rEa4iM1y4O5qCMpYxdiypykyf9yp7C81cJTZHKMOvrnGcZiqxlMfyQsaUUA=="
+  }
+}
+```
+
+| Value type          | string (crypto package asymmetric encryption algorithms) |
+|:--------------------|:---------------------------------------------------------|
+| **Possible values** | `"tendermint/PrivKeyEd25519"`                            |
+|                     | `"tendermint/PrivKeySecp256k1"`                          |
+|                     | `"tendermint/PrivKeySr25519"`                            |
+
+The string values are derived from the asymmetric cryptographic implementations defined in the `crypto` package.
+
+CometBFT will always generate an Ed25519 key-pair for node ID using the `cometbft init` or the `cometbft gen-node-key`
+command. Other types of encryption keys have to be created manually. (See examples under
+[priv_key.value](#priv_keyvalue).)
+
+### priv_key.value
```

Comment:
Are you sure we accept  different crypto algorithms here?

Reply:
Yup, tested, it works for node IDs. They are very simple identifiers. The algorithm is only used to create the node ID (wallet address) but then it's not used for anything else.